parkings/migrations/0029_enforcement_domain.py

```diff
@@ -14,16 +15,57 @@ class Migration(migrations.Migration):
     ]
 
     operations = [
+        migrations.CreateModel(
+            name='EnforcementDomain',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('created_at', models.DateTimeField(auto_now_add=True, verbose_name='time created')),
+                ('modified_at', models.DateTimeField(auto_now=True, verbose_name='time modified')),
+                ('code', models.CharField(max_length=10, unique=True, verbose_name='code')),
+                ('name', models.CharField(max_length=40, verbose_name='name')),
+                ('geom', django.contrib.gis.db.models.fields.MultiPolygonField(
+                    srid=3879, verbose_name='geometry', default="MULTIPOLYGON EMPTY"
+                )),
+            ],
+            options={
+                'ordering': ('code',),
+            },
+        ),
+        migrations.CreateModel(
+            name='Enforcer',
+            fields=[
+                ('created_at', models.DateTimeField(auto_now_add=True, verbose_name='time created')),
+                ('modified_at', models.DateTimeField(auto_now=True, verbose_name='time modified')),
+                ('id', models.UUIDField(default=uuid.uuid4, editable=False, primary_key=True, serialize=False)),
+                ('name', models.CharField(max_length=80, verbose_name='name')),
+                ('enforced_domain', models.ForeignKey(help_text='An enforcement domain than is enforced by this enforcer', null=True, on_delete=django.db.models.deletion.PROTECT, to='parkings.EnforcementDomain', verbose_name='enforced domain')),
+                ('users', models.ForeignKey(null=True, on_delete=django.db.models.deletion.PROTECT, to=settings.AUTH_USER_MODEL, verbose_name='users')),
+            ],
+            options={
+                'verbose_name': 'enforcer',
+                'verbose_name_plural': 'enforcers',
+            },
+        ),
         migrations.AddField(
             model_name='paymentzone',
             name='code',
             field=models.CharField(max_length=10, null=True),
         ),
         migrations.AddField(
             model_name='permitseries',
-            name='owner',
+            name='owner_operator',
             field=models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.PROTECT, related_name='permit_series_set', to='parkings.Operator'),
         ),
+        migrations.AddField(
+            model_name='permitseries',
+            name='owner_enforcer',
+            field=models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.PROTECT, related_name='permit_series_set', to='parkings.Enforcer'),
+        ),
+        migrations.AlterField(
+            model_name='paymentzone',
+            name='code',
+            field=models.CharField(max_length=10),
+        ),
```

Comment:
Move these changes to the `code` field up to the `AddField`-statement of the `code` field to avoid adding the field and changing it in same migration.